app/scenes/Invite.tsx

```diff
@@ -209,8 +209,8 @@ function Invite({ onSubmit }: Props) {
                   placeholder={`name@${predictedDomain}`}
                   value={invite.email}
                   required={index === 0}
-                  autoComplete="off"
                   autoFocus
+                  autoComplete="new-email"
```

Comment:
Where are you seeing that this is a valid value? It's not listed here for example:

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/autocomplete

Reply:
Got it, we don't want to include a non-standard value and undocumented in that case.